doc/devel/content_guide.rst

```diff
@@ -0,0 +1,273 @@
+.. _documenting-content:
+
+******************
+Content guidelines
+******************
+
+The following content guidelines aim to improve the consistency, cohesiveness, and
+organization of the documentation on using the library. Broadly, this documentations
+falls into one of these sections:
+
+:ref:`plot_types`
+    Summary of visualization techniques implemented as high-level API
+
+:ref:`users-guide-explain`
+    | Explanations of key concepts and core functionality,
+    | e.g. :ref:`what-is-a-backend`, :ref:`colors_def`, and  :ref:`event_handling`
+
```

Comment:
I'd suggest something much closer to:

```suggestion
:ref:`users-guide-explain`
    The users guide to Matplotlib, meant to be read in order with enough detail and examples 
    for users to be able to use Matplotlib effectively.  
```

I would consider the three examples you give to be considerable asides to the Users Guide, rather than representative of their main goal.  The previous users guide had things like that as the _only_ things in it, but that just made it a grab bag of disconnected subjects that no one would call a "User's Guide"